utils/RunONNXModelZoo.py

```diff
@@ -315,8 +319,10 @@ def check_model(model_path, model_name, compile_args, report_dir):
         logger.debug("[{}] {}".format(model_name, msg))
 
         if args.Html:
+            from datetime import datetime
```

Comment:
Move this to the top in front of joblib import?

Reply:
Done.